examples/README.md

```diff
@@ -86,6 +86,30 @@ $ PYTHONPATH=../ ../krun.py example.krun
 You should see a log scroll past, and results will be stored in the file:
 `../krun/examples/example_results.json.bz2`.
 
+## Running in reboot and resume modes
+
+krun can resume an interrupted benchmark by passing in the `--resume` flag:
+
+```bash
+$ PYTHONPATH=../ ../krun.py --resume example.krun
+```
+
+You may wish to use this facility to reboot after every execution.
+To do this, you can pass in both the `--resume` and `--reboot` flags when
+you start krun:
+
+```bash
+$ PYTHONPATH=../ ../krun.py --resume --reboot example.krun
+```
+
+You will also need to ensure that krun is restarted once the machine has
+rebooted.
+You can do this by hand, or by using the boot configuration provided by
+your OS.
+The `krun/etc` directory contains an `rc.local` file which goes with the
```

Comment:
Can we rename it `rc.local.linux` as an example `rc.local` will follow for OpenBSD too.